package.json

```diff
@@ -22,5 +22,8 @@
     "lint": "yarn workspaces foreach -pt run lint",
     "test": "yarn workspaces foreach -pt run test"
   },
-  "packageManager": "yarn@3.5.1"
+  "packageManager": "yarn@3.5.1",
+  "dependencies": {
+    "ethers": "^6.7.1"
```

Comment:
Curious: why was this added?

Reply:
Good catch. Removed in https://github.com/statechannels/go-nitro/pull/1789/commits/c92fcf19641294135ba12cb8930fef100e3f62b0. It was added by mistake